packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -272,10 +301,35 @@ export class VaTextInput {
         'usa-character-count__status': charcount,
         'usa-character-count__status--invalid': charcount && maxlength && value?.length > maxlength
       });
+
+      const isFormsPattern = useFormsPattern === 'single' || useFormsPattern === 'multiple' ? true : false;
+
+      let formsHeading = null;
+      if (isFormsPattern) {
+        const HeaderLevel = getHeaderLevel(formHeadingLevel);
+        formsHeading = (
+          <Fragment>
+            {formHeading &&
+              <HeaderLevel id="form-question" part="form-header">
+                {formHeading}
+              </HeaderLevel>
+            }
+            {formDescription && 
+              <div id="form-description" class="usa-legend" part="form-description">{formDescription}</div>
```

Comment:
I have added `part` to both the form header and form description in case teams need to customize those styles for whatever reason. I can revert though if we feel like there's not really a need for it.